### PR TITLE
Implement Controller class

### PR DESCRIPTION
**Pull request by tanabahh**
Класс который парсит файл конфигурации, ставит задачи на выполнение с помощью apscheduler

### CODE REVIEW

**Review by nikmel2803 — APPROVED** [review]

File: src/taskscontroller.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+class TasksController:
+    """A class that parses a configuration file,
+    sets tasks to be executed using apscheduler"""
+
+    def get_tasks(self) -> Union[List[Tuple[TaskWrapper, Dict]], None]:
+        """Parses the task file, wraps it in a wrapper
+        class, returns a list with
+        tuples that contain the TaskWrapper and schedule"""
+
+        with open(TASKS_FILE_PATH, "r") as file:
```
Comment (n0str) [inline src/taskscontroller.py]: `file` is ok for variable name, however several editors highlight it specifically. confuses...

Comment (n0str) [inline src/taskscontroller.py]: `with open` can raise an exception. need to handle this

File: src/taskscontroller.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+logger = getLogger("general")
+scheduler = BlockingScheduler()
+scheduler.configure(**SCHEDULER)
+
+
+class TasksController:
+    """A class that parses a configuration file,
+    sets tasks to be executed using apscheduler"""
+
+    def get_tasks(self) -> Union[List[Tuple[TaskWrapper, Dict]], None]:
```
Comment (n0str) [inline src/taskscontroller.py]: `get_tasks` return `List` of `Tuple[TaskWrapper, Dict]`. But semantically it should be a `List of Tasks`. Maybe to create a class or type or dict for this kind of structure?

it is difficult to understand (why tuple? What type is the first element, etc):
```
 tasks.append((TaskWrapper(**src), schedule))
```

File: src/taskscontroller.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        TaskWrapper.notifiers = src_notifiers
+        return tasks
+
+    def process(self, serialized_task: dict):
+        """Runs scenario
+
+        :param serialized_task: dict consisting of primitive types
+        """
+
+        task: TaskWrapper = TaskWrapper.deserialize(serialized_task)
```
Comment (n0str) [inline src/taskscontroller.py]: is it necessary? IDE can't understand that `deserialize` return TaskWrapper?

**Review by n0str — APPROVED** [review]

**Pull request merged**